src/tic_tac_toe/user_interface.clj

```diff
@@ -48,10 +48,10 @@
       (.update-display this)))
 
   (prompt-selection
-    [this prompt]
+    [this func prompt]
     (.clear-src this)
     (.update-display this prompt)
-    (.get-input this))
+    (func this))
```

Comment:
I no longer follow what's happening in this function, is there a way to clean this up to make it more people readable?

Reply:
I debated this change.  I wanted to make something DRYer, but wasn't happy with what I did.

I've reverted prompt-selection to its HEAD state, and modified [exit](https://github.com/saralein/tic-tac-toe_clojure/pull/13/files#diff-52c69c82bcdbcc16e3156d4a9033a37cR70).  Thank you!